### PR TITLE
Feature/139563053/automate reloading on dev

### PR DESCRIPTION
**Pull request by iAmao**
#### What does this PR do?
This PR automates browser reloading.
#### Description of Task to be completed?
Browser reloads when changes are made to either the html, css or js file
#### How should this be manually tested?
* npm install
* gulp runserver
#### Any background context you want to provide?
N/A
#### Video
https://www.youtube.com/watch?v=DtYP-6Cjf3Y&feature=youtu.be
Questions:
N/A

### CODE REVIEW

**Comment by victor-enogwe** [comment]
nice! looks good to me

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
 An Inverted Index is an Index Data Structure storing a mapping from content to location (i.e. a mapping of words or numbers to it's location in a database file)
+
+
+### Automate Development
+
+To reload browser automatically when changes are detected in html, css or js file while in development, you need to :
+* Clone repository
+* run : npm install -g gulp browser-sync
+* run : npm install
+* run : gulp runserver
```
Comment (iAmao) [inline README.md]: Make readme file a little bit more explanatory

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const gulp = require('gulp'),
+      browserSync = require('browser-sync');
```
Comment (iAmao) [inline gulpfile.js]: Declare browserSync with const

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+gulp.task('serve', function () {
```
Comment (iAmao) [inline gulpfile.js]: Use arrow function

File: gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      baseDir: './dist'
+    }
+  });
+});
+
+gulp.task('watch-html', browserSync.reload);
+
+
+// Default task
+gulp.task('runserver', ['serve', 'watch-html'], function() {
```
Comment (iAmao) [inline gulpfile.js]: Use arrow function

**Pull request merged**